cmake/modules/FindICU.cmake

```
@@ -0,0 +1,40 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Custom FindICU.cmake
+# Create imported targets even if libraries are not available yet
+
+if(NOT TARGET ICU::uc)
+    add_library(ICU::uc UNKNOWN IMPORTED)
+    set_target_properties(ICU::uc PROPERTIES
+        IMPORTED_LOCATION ${ICU_UC_LIB_RELEASE}
+        IMPORTED_LOCATION_DEBUG ${ICU_UC_LIB_DEBUG}
+    )
+endif()
+if(NOT TARGET ICU::i18n)
+    add_library(ICU::i18n UNKNOWN IMPORTED)
+    set_target_properties(ICU::i18n PROPERTIES
+        IMPORTED_LOCATION ${ICU_I18N_LIB_RELEASE}
```

```suggestion
        IMPORTED_LOCATION_RELEASE ${ICU_I18N_LIB_RELEASE}
```
why not explicit `RELEASE`?